Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,108 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  Git_URL=""
+  Git_SHA=""
+  Ruby_URL=""
+  Ruby_SHA=""
+else
+  Git_URL=""
+  Git_SHA=""
+  Ruby_URL=""
+  Ruby_SHA=""
+fi
+
+cleanup-old() {
+  for version in "$HOMEBREW_LIBRARY/Vendor/$VENDOR_NAME/Versions"/*
```

Comment:
`Library/Homebrew/vendor/ruby/2.x.y` would be my favourite choice. It's not uncommon that you want to do a `grep` with tools that allow you to exclude folders and this would allow excluding just `Library/Homebrew/vendor` rather than both `Library/Homebrew/vendor` and `Library/Vendor`.